include/box2d/b2_contact.h

```diff
@@ -350,4 +351,7 @@ inline float b2Contact::GetTangentSpeed() const
 	return m_tangentSpeed;
 }
 
+/// internal for testbed, do not use
+extern B2_API bool g_blockSolve;
```

Comment:
I think it would be cleaner to add `B2_API` to the declaration.

Reply:
Yes, good idea! It will be better.